jujugui/static/gui/src/app/views/state.js

```diff
@@ -144,7 +144,8 @@ YUI.add('juju-app-state', function(Y) {
       var sections = {
         app: 'App',
         sectionA: 'SectionA',
-        sectionB: 'SectionB'
+        sectionB: 'SectionB',
+        sectionC: 'SectionC'
       };
       // calls _dispatchSectionA or _dispatchSectionB
```

Comment:
sure I can update that, was just an oversight when doing all these mechanical changes.